pkg/config/validation/validation.go

```diff
@@ -3164,6 +3157,9 @@ func validateLocalityLbSetting(lb *networking.LocalityLoadBalancerSetting) error
 
 	srcLocalities := make([]string, 0, len(lb.GetDistribute()))
 	for _, locality := range lb.GetDistribute() {
+		if locality.From == "" || locality.To == nil {
+			return fmt.Errorf("locality lb distribute settings must specify both From and To")
```

Comment:
I don't think this is required. Its not documented in the API and its possible to configure it this way without issue.

Reply:
BTW, I have opened https://github.com/istio/api/pull/2061 to clarify whether we should accept changes like this. It would be good to get consensus there first.

It is a tradeoff - but I think we should favor backwards compatibility over stricter validation.

Reply:
ok